src/KMonad/Prelude/Imports.hs

```diff
@@ -0,0 +1,38 @@
+{- The very first module that gets imported, only does external imports -}
+module KMonad.Prelude.Imports
+  ( module X )
+where
+
+import Control.Arrow           as X (left, right)
+import Control.Exception       as X (throw)
+import Control.Exception.Lens  as X
+import Control.Lens            as X
+import Control.Monad.Cont      as X
+import Data.Acquire            as X
+import GHC.Conc                as X (orElse)
+import RIO.Text                as X (unlines, lines, unpack, pack)
+
+import RIO as X hiding
+  (-- Not the lens stuff, I want more support for lenses from "Control.Lens"
+    view, ASetter, ASetter', Lens, Getting, Lens'
+  , SimpleGetter, lens, over, set, sets, to, (^.)
+
+    -- The following line is required for newer stack releases.
+    -- This is also the reason for the OPTIONS_GHC pragma
+    -- , (^..), (^?), preview, (%~), (.~)
```

Comment:
Which `OPTIONS_GHC` pragma?  For what it's worth, this tripped me up for a second because I tried compiling with GHC 8.10.4; at least it was documented how to fix it :)